src/openforms/submissions/tasks/emails.py

```diff
@@ -60,7 +66,18 @@ def send_confirmation_email(submission_id: int) -> None:
     So, if we're really having a lot of bad luck, a race condition is possible and then
     the confirmation e-mail would be sent out twice, which is an acceptable risk.
     """
-    submission = Submission.objects.get(id=submission_id)
+    try:
+        submission = Submission.objects.select_for_update(nowait=True).get(
+            id=submission_id
+        )
+    except DatabaseError:
+        logger.debug(
+            "Submission %d confirmation e-mail task failed to acquire a lock. This is "
```

Comment:
Since this prevents the race condition, should the comment at the beginning of the function be changed?